### PR TITLE
Move videopress options

### PR DESCRIPTION
**Pull request by leogermani**
#### Changes proposed in this Pull Request:

* Migrates VideoPress options to the package

#### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?

#### Jetpack product discussion

pe4Cmx-40-p2
#### Does this pull request change what data or activity we track or use?

No

#### Testing instructions:

* Go to Jetpack > Settings
* Search for videopress
* tweak videopress options
* Make sure all works fine (options are persisted)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202561463041776

### CODE REVIEW

File: projects/plugins/jetpack/modules/videopress/class.videopress-ajax.php [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <?php // phpcs:ignore WordPress.Files.FileName.InvalidClassFileName
 
 use Automattic\Jetpack\Connection\Client;
+use Automattic\Jetpack\VideoPress\Options as VideoPress_Options;
```
Comment (fgiannar) [inline projects/plugins/jetpack/modules/videopress/class.videopress-ajax.php:4]: Given `VideoPress_Options` is defined in a separate file (`class.videopress-options.php`) this should be fine 👍

File: projects/plugins/jetpack/modules/videopress/class.videopress-options.php [hunk 2]
```diff
@@ -5,55 +5,40 @@
  * @package automattic/jetpack
  */
 
+use Automattic\Jetpack\VideoPress\Options as Package_Options;
```
Comment (fgiannar) [inline projects/plugins/jetpack/modules/videopress/class.videopress-options.php:8]: Is this whole file going to be removed in a future version?

**Review by fgiannar — COMMENTED** [review]
Looks good in general, I'm only a bit confused on the context: In the testing instructions you mention enabling/disabling the module and tweaking the video privacy option.
However, this option is unrelated to the changes here, right?

From what I can tell, there are currently no videopress options stored in the DB. Only the logic to return the `shadow_blog_id` in case the plan supports Videopress.

I tested the patch on a site with/without the Videopress plan and ensured that it works as expected!

If I got things right, I believe this is good to go!

File: projects/plugins/jetpack/modules/videopress/class.videopress-ajax.php (hunk 1 above)
Comment (leogermani) [reply projects/plugins/jetpack/modules/videopress/class.videopress-ajax.php:4]: Yes, I don't include that file and name the new class as the rest of the code expects... I think it's fine. But if you think it's safer to rename it I'd do it

**Comment by leogermani** [comment]
> From what I can tell, there are currently no videopress options stored in the DB. Only the logic to return the shadow_blog_id in case the plan supports Videopress.

hmm.. right... I didn't look it through and I was under the impression that the Privacy option was stored there... apparently it's not. There's nothing there...

File: projects/plugins/jetpack/modules/videopress/class.videopress-options.php (hunk 2 above)
Comment (leogermani) [reply projects/plugins/jetpack/modules/videopress/class.videopress-options.php:8]: Yes. again only keeping it there to be extra safe

**Review by fgiannar — DISMISSED** [review]

**Review by fgiannar — APPROVED** [review]

**Pull request merged**